diff --git a/maude_hcs/lib/network/net-dns_prob.maude b/maude_hcs/lib/network/net-dns_prob.maude
index 2a13049e..aa5e381f 100644
--- a/maude_hcs/lib/network/net-dns_prob.maude
+++ b/maude_hcs/lib/network/net-dns_prob.maude
@@ -108,7 +108,8 @@ eq mkPubDnsNetS(myAddr,pubDnsAddr,ixpAddr) =
   /\ dixp := delayParamdX(j,msg_to_ixp)  
   /\ dtgt := delayParamdX(s j,msg_to_pub)  
   /\ tmsg := tm(T,T + dixp, T + dtgt, msg_to_pub)
-  /\ advAttrs1 := addTM2Dns(advAttrs,tmsg)
+  --- /\ advAttrs1 := addTM2Dns(advAttrs,tmsg)
+  /\ advAttrs1 := advAttrs
   /\ dmsg := [dtgt,msg_to_net,0]
  .  
 
@@ -198,7 +199,8 @@ ceq  netDnsSrvResponse(netAttrs,advAttrs,netAddr,amsg,j)
   /\ dixp := delayParamdX(j,msg_to_ixp)  
   /\ dtgt := delayParamdX(s j,msg_to_loc)  
   /\ tmsg := tm(T,T + dixp, T + dtgt, msg_to_loc)
-  /\ advAttrs1 := addTM2Dns(advAttrs,tmsg)
+  --- /\ advAttrs1 := addTM2Dns(advAttrs,tmsg)
+  /\ advAttrs1 := advAttrs
   /\ dmsg := [dtgt,msg_to_loc,0]
 .   
 
diff --git a/maude_hcs/lib/network/net_prob.maude b/maude_hcs/lib/network/net_prob.maude
index 887f009c..2163ab63 100644
--- a/maude_hcs/lib/network/net_prob.maude
+++ b/maude_hcs/lib/network/net_prob.maude
@@ -197,7 +197,8 @@ ceq [netClientGenTcp]:
     /\ netAttrs1, connectionCount: ctr, newCnx: b := netAttrs
     /\ tpl := genTcpTPL(T,msg,TIME_LIST,TIME_LIST_TO_IXP, 
                         size_list, ctr, b) 
-    /\ advAttrs1 := addTPL2tcp(advAttrs,tpl)  
+    --- /\ advAttrs1 := addTPL2tcp(advAttrs,tpl)  
+    /\ advAttrs1 := advAttrs
     /\ netAttrsNext0, pending: amsgs := netAttrsNext
     
     [ print "(" T ") netClientGenTcp " netAddr " sending " transportMsg " with delay " delay:Float  ]
@@ -342,7 +343,8 @@ eq qrMatch(Q:Query,R:Response) = false [owise] .
    /\ flowAttrs0, flowId: ctr  := flowAttrs
    /\ tpl := genTcpTPL(T,msg,TIME_LIST,TIME_LIST_TO_IXP, size_list,
                        ctr,false) 
-    /\ advAttrs0 := addTPL2tcp(advAttrs,tpl)   
+    --- /\ advAttrs0 := addTPL2tcp(advAttrs,tpl)   
+    /\ advAttrs0 := advAttrs
 **** I thought this was removed?    
     /\ netAttrs1 := removeConnection(netAttrs,REMOTE_ADDR) 
     [ print "(" T ") Net server " netAddr " handling response, sending " transportMsg "." ]
